### ISSUE TITLE
Feature: Procesamiento de datos

### ISSUE THREAD

**Issue by gemayc** [Bootcamp-IA-P6/project-6-team-4-ml#7]
Descripción:
Cada integrante prepara los datos según las conclusiones de SU propio EDA.
⚠️ IMPORTANTE - Común para TODOS:

 Train/Test Split: MISMO para todos (80/20, random_state=42, stratify=y)

python  X_train, X_test, y_train, y_test = train_test_split(
      X, y, test_size=0.2, random_state=42, stratify=y
  )
⚠️ IMPORTANTE - Estructura de archivos:

NO crear notebook separado de preprocesamiento
El preprocesamiento va en el MISMO notebook del modelado
Solo 1 notebook por persona: notebooks/[iniciales]_modelo_completo.ipynb

Tareas individuales (cada uno decide según su EDA):

 Tratamiento de outliers: Basado en tu análisis

¿Los eliminas, imputas o mantienes?
Justificar decisión

 Tratamiento de valores cero en Colesterol/Presión:

¿Los consideras como nulos?
¿Los imputas (media, mediana)?
¿Los dejas?

 Encoding de categóricas:

Label Encoding
One-Hot Encoding
Tu elección según el algoritmo

 Escalado:

StandardScaler
MinMaxScaler
RobustScaler
Tu elección

 Feature Engineering (opcional):

Crear nuevas variables si lo crees necesario
Interacciones entre variables
Transformaciones

 Balanceo de clases (opcional):

SMOTE
class_weight
Ninguno

Entregables por persona:
✅ Todo en 1 solo notebook que contenga:

Sección de preprocesamiento con justificaciones
(Opcional) Guardar scaler/encoders si necesitas reutilizarlos
